types/pouchdb-live-find/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for pouchdb-live-find 0.4
+// Project: https://github.com/colinskow/pouchdb-live-find
+// Definitions by: assemblethis <https://github.com/assemblethis>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+/// <reference types="pouchdb-core" />
+declare namespace PouchDB {
+    namespace LiveFind {
+        interface RequestDef<Content extends {}> extends Pick<Find.FindRequest<Content>, Exclude<keyof Find.FindRequest<Content>, 'use_index'>> {
+            aggregate?: boolean; // if true outputs an aggregate list on every update event
+        }
+
+        interface PaginateOptions<Content extends {}> extends Pick<Find.FindRequest<Content>, 'sort' | 'skip'| 'limit'> {}
```

Comment:
I think that declare an enum for 'sort' | 'skip'| 'limit' will be helpful.

Reply:
These are the keys of the interface that I'm picking. Can you give an example of how you would handle this with an enum?

Reply:
What about something like https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5344bfc80508c53a23dae37b860fb0c905ff7b24/types/pinterest-sdk/index.d.ts#L13